middleware/lib/sfMiddleware.ts

```diff
@@ -97,7 +95,12 @@ function createOrg(logger: Logger, reqContext: any, accessToken?: string): Org {
         throw new Error(message);
     }
 
-    const apiVersion = reqContext.apiVersion || process.env.FX_API_VERSION || Constants.CURRENT_API_VERSION;
+    const apiVersion = reqContext.apiVersion || process.env.FX_API_VERSION;
+    if (!apiVersion) {
```

Comment:
We need an API version and cannot assume a version.

Reply:
Where does `reqContext.apiVersion` come from? The payload or from the sf org?

Reply:
Correct.

Reply:
nice.